lib/loader/ast/instruction.cpp

```diff
@@ -234,21 +234,23 @@ Expect<void> Loader::loadInstruction(AST::Instruction &Instr) {
 
   case OpCode::Br_table: {
     uint32_t VecCnt = 0;
-    // Read the vector of labels.
-    if (auto Res = readU32(VecCnt); unlikely(!Res)) {
-      return Unexpect(Res);
-    }
-    if (VecCnt / 2 > FMgr.getRemainSize()) {
-      // Too many label for Br_table.
```

Comment:
You also deleted the reason why we have to check the value here.

Reply:
I think this comparison with `VecCnt / 2` with `FMgr.getRemainSize()` is a general error, and we don't have to specially include this comment here in the code.